.github/workflows/version-bump.yml

```diff
@@ -1,13 +1,12 @@
 ---
-name: Bump version
-run-name: Bump version to ${{ inputs.version_number }}
+name: Version Bump
 
 on:
   workflow_dispatch:
     inputs:
-      version_number:
-        description: "New version (example: '2024.1.0')"
-        required: true
+      version_number_override:
+        description: "New version override (leave blank for automatic calculation, example: '2024.1.0')"
+        required: false
```

Comment:
⛏️  It would be nice to have some validation on this input, simply to ensure that we fail to version bump if the version number provided isn't what we expect.

Reply:
I have an action I created for input validation [here](https://github.com/bitwarden/gh-actions/pull/262). Let me know what you think.